src/main/java/com/andrew/strings/StringUtilsImpl.java

```diff
@@ -0,0 +1,88 @@
+package com.andrew.strings;
+
+import java.util.ArrayList;
+
+public class StringUtilsImpl implements StringUtils {
+    @Override
+    public int charCount(String str, char letter) {
+        int count = -1;
+        char[] arrayStr = str.toCharArray();
+
+        for (int i = 0; i < arrayStr.length; i++) {
+            if (arrayStr[i] == letter)
+                count++;
+        }
+        return count;
+    }
+
+    @Override
+    public String reverse(String str) {
+        char[] arrayStr = str.toCharArray();
+        for (int i = 0; i < arrayStr.length / 2; i++) {
+            char letter = arrayStr[i];
+            arrayStr[i] = arrayStr[arrayStr.length - i - 1];
+            arrayStr[arrayStr.length - i - 1] = letter;
+        }
+        String string = new String(arrayStr);
```

Comment:
do you actually need  inline variable `string` here?

Reply:
I really need it because i must transform array of char into String